easybuild/easyconfigs/r/RSEM/RSEM-1.2.31-goolf-1.4.10.eb

```diff
@@ -0,0 +1,47 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Pablo Escobar Lopez
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics 
+
+easyblock = 'ConfigureMake'
+
+name = "RSEM"
+version = "1.2.31"
+
+homepage = 'http://deweylab.biostat.wisc.edu/rsem/'
+description = """ RSEM is a software package for estimating gene and isoform expression 
+ levels from RNA-Seq data. The RSEM package provides an user-friendly interface, supports 
+ threads for parallel computation of the EM algorithm, single-end and paired-end read data, 
+ quality scores, variable-length reads and RSPD estimation. """
+
+toolchain = {'name': 'goolf', 'version': '1.4.10'}
+
+source_urls = ['https://github.com/deweylab/RSEM/archive/']
+sources = ['v%(version)s.tar.gz']
+
+unpack_options = '--strip-components=1'
+
+# RSEM will use bowtie / bowtie2 / STAR if found in user $PATH
+# add them as dependencies if required
+#dependencies = [
+#    ('Bowtie', '1.0.0'),
```

Comment:
wouldn't it be better to add those deps, to avoid that anything from the OS is picked up?